### PR TITLE
VOPR minor quality of life improvements

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

**Review by sentientwaffle — DISMISSED** [review]

**Comment by sentientwaffle** [comment]
Ah looks like `fmt` failed

File: build.zig [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ const JniTestStep = struct {
                 else if (std.mem.indexOf(u8, ldd_result, "libc") != null)
                     .gnu
                 else {
-                    std.log.err("{s}", .{ ldd_result });
+                    std.log.err("{s}", .{ldd_result});
```
Comment (matklad) [inline build.zig:1021]: @cb22 hm, haven't we fixed `zig fmt` not gating the main branch? This is pre-existing I think?

**Review by sentientwaffle — APPROVED** [review]

File: build.zig (hunk 1 above)
Comment (cb22) [reply build.zig:1021]: Hmmm, it only runs on `src/`

Comment (sentientwaffle) [reply build.zig:1021]: It looks like we only check the zig files in `src/`, so `build.zig` itself is not checked.

**Pull request merged**